docs/topics/gradle/gradle-compiler-options.md

```diff
@@ -40,21 +40,26 @@ import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
 val compileKotlin: KotlinCompile by tasks
```

Comment:
```suggestion
val compileKotlin: KotlinCompilationTask<*> by tasks
```

Reply:
And the import above should be org.jetbrains.kotlin.gradle.tasks.KotlinCompilationTask?